### PR TITLE
Fix CommandPermissions fetch

### PR DESCRIPTION
**Pull request by xWildFire**
:command_id and :application_id not defined

### CODE REVIEW

**Comment by CommandString** [comment]
There is command_id property specified in the [Application Command Permissions Structure Docs](https://discord.com/developers/docs/interactions/application-commands#application-command-permissions-object-guild-application-command-permissions-structure). Could I ask what you're trying to accomplish?

**Comment by xWildFire** [comment]
> Could I ask why this is a bug? Additionally, could you provide the code that you're using in combination with this _bug fix_.

`$interaction->guild->command_permissions?->fetch($commandId);`

without fix(missing application_id and command_id):

`[04.02.2023, 15:35:09] [WARNING] REQ GET applications/:application_id/guilds/484723871887392772/commands/:command_id/permissions failed: Discord\Http\Exceptions\RequestFailedException: Bad Request - {
    "code": 50035,
    "errors": {
        "application_id": {
            "_errors": [
                {
                    "code": "NUMBER_TYPE_COERCE",
                    "message": "Value \":application_id\" is not snowflake."
                }
            ]
        },
        "command_id": {
            "_errors": [
                {
                    "code": "NUMBER_TYPE_COERCE",
                    "message": "Value \":command_id\" is not snowflake."
                }
            ]
        }
    },
    "message": "Invalid Form Body"
}`

File: src/Discord/Repository/Guild/CommandPermissionsRepository.php [hunk 1]
```diff
@@ -45,4 +45,14 @@ class CommandPermissionsRepository extends AbstractRepository
      * {@inheritDoc}
      */
     protected $class = CommandPermissions::class;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function __construct(Discord $discord, array $vars = [])
+    {
+        $vars['application_id'] = $discord->application->id; // For the bot's Application Guild Commands
```
Comment (Exanlv) [inline src/Discord/Repository/Guild/CommandPermissionsRepository.php:54]: I'm not sure of what the plan is for the PR but regardless of that, this comment is pretty meaningless 😄

Comment (xWildFire) [reply src/Discord/Repository/Guild/CommandPermissionsRepository.php:54]: it's not written by me, copied from another dphp file :D

**Review by SQKo — APPROVED** [review]
While i still don't remember why i didn't change this, it seems wont be a problem to merge, we'll see later if it has problem.

**Pull request merged**